actors/miner/src/lib.rs

```diff
@@ -1832,6 +1835,9 @@ impl Actor {
         // Activate data for proven sectors.
         let (data_batch, data_activations) =
             activate_sectors_pieces(rt, data_activation_inputs, params.require_activation_success)?;
+        if data_batch.success_count == 0 {
+            return Err(actor_error!(illegal_argument, "all data activations failed"));
```

Comment:
After removing the proofs check, this message can be just "all activations failed".